Makefile

```diff
@@ -37,7 +37,6 @@ protoc-local:
 
 lint:
 	golangci-lint run
-	go vet ./...
 	# Uncomment this once we update to go1.23 which makes the -diff flag available.
 	# See https://tip.golang.org/doc/go1.23#go-command
 	# go mod tidy -diff
```

Comment:
how about we uncomment this line that I forgot to uncomment and also add it to CI?

Reply:
Ok with me doing this in a followup PR, to avoid doubling up on PR purpose?